lib/src/fnc/script/modules/surrealdb/functions/type/is.rs

```diff
@@ -0,0 +1,31 @@
+use super::run;
+use crate::fnc::script::modules::impl_module_def;
+
+pub struct Package;
+
+impl_module_def!(
+	Package,
+	"type::is",
```

Comment:
This will cause the javascript module to be called `type::is` and you would have to do `surrealdb.functions.type["type::is"].function_name` instead of `surrealdb.function.type.is.function_name` to access the function.

Reply:
Seems like some I hadn't noticed that there are a bunch of existing exported functions which make the same mistake like `vector::distance`. will have to fix that.